change/@ni-nimble-components-8ca8762f-ad08-436d-980f-56d9f5ba9e39.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You are going to hate me: I'm seeing some clunky behavior in Safari. 

When scrolling slowly, I still see flickers of row highlighting. Similar when using momentum scroll as the scroll slows down, but it's even worse because the flicker occurs over a row that I'm not hovered on.

You can see both effects in this video (first tab is the build from this PR, second tab is the public storybook).

Edge and Firefox on macOS both work great.

https://github.com/user-attachments/assets/fa59ec76-7619-45d3-bc1f-13b0929e1509

Reply:
Looking a lot better! I've only noticed one issue so far: there was one time that a row got stuck in the hovered state even though the mouse wasn't over it. If I hovered other rows, I ended up with two rows showing the hover state.

I think it happened when I had right clicked on the table to open dev tools in a new window. When I later returned to the primary browser window it was in that state. But I haven't been able to reproduce those conditions again; I'll keep trying.

This isn't fatal or anything: just hovering over the row that was stuck was sufficient to put it back in a good state.

Reply:
That doesn't sound like it would be related to the original problem or my change. I don't think there's any way for our hover styling to apply unless the browser thinks it is hovered (i.e. the `:hover` pseudo-state applies). My changes could only _reduce_ the cases where hover styling is applied, and I can't affect whether the browser does or doesn't register that an element is being hovered. My guess is we'd be equally likely to reproduce what you saw in `main`. Based on that reasoning, I'm going to mark this thread resolved.